src/lib/colourType.js

```diff
@@ -12,5 +12,5 @@ export const isHex = colour => {
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
 export const isKeyword = colour => {
-    return ~Object.keys(colourNames).indexOf(colour.toLowerCase());
```

Comment:
it would be better to use `!!Object.getOwnPropertyDescriptor(colourNames, colour.toLowerCase())`.

`in` checks the entire prototype chain, and you don't want that.